tools/system_libs.py

```diff
@@ -1835,6 +1835,11 @@ def build_port(port_name, settings):
     port.get(Ports, settings, shared)
 
 
+def clear_port(port_name, settings):
+  port = ports.ports_by_name[port_name]
+  port.clear(Ports, settings, shared)
```

Comment:
Does ports' `clear` function need `Ports` (1st argument)? It doesn't seem to be used..

Reply:
I'm not sure, perhaps there are no ports that use it, perhaps it can be removed but maybe as a followup to this PR?

Reply:
That sounds good too